### PR TITLE
Kejing Li - hw01

### PR DESCRIPTION
**Pull request by Mirrorlkj**
This should include everything asked for: Introduction, R markdown Syntax & reflection! 

hope this works out!

### CODE REVIEW

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* It is great practice to commit regularly like you did.
* Great job on completing all requirements for the HW :)
* Just for future reference, reflection should be on the comment section when you pull request.
* I love your English name!!

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |           |       X       |            |

Remarks:

* It is great practice to commit regularly like you did.
* Great job on completing all requirements for the HW :)
* Just for future reference, reflection should be on the comment section when you pull request.
* I love your English name!!
* Make sure you commit files through RStudio, not uploading through github. This will result in the reduction of point in the ease-of-access score in subsequent assignments.

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+  - Psy
+  - QMSA
+- **Home**: Hangzhou, China
+
+## Website
+
+[My Github Page](https://github.com/Mirrorlkj)
+
+## Pic
+- [x] __This is my profile photo!__
```
Comment (wanitchayap) [inline README.md:15]: This is a neat icon!

File: README.md [hunk 2]
```diff
@@ -1 +1,24 @@
...
+
+![Profile Photo](https://raw.githubusercontent.com/Mirrorlkj/hw01/master/IMG_7482.JPG)
+
+## Reflection and Experience 
+- **Problem 0:** no previous experience with any programming language, first study in an English enviornment. The process is the opposite of painless and takes me loooooooonger time to figure everything out.
+
+- **Problem 1:** R punched me at the very beginning with an unreadale error message when I tried to push/pull. I installed every components but there was nothing in the upper right pane under git tab. Neither TA and teacher knew what was going on, just when I was thinking about using R cloud instead. My roommate suggested I could try to change the file name from chinese to english. Finally, I changed the computer system's priority language and it worked out !!! 
+
+**Everything else could be found on our useful website! _(it just takes time)_**
```
Comment (wanitchayap) [inline README.md:24]: Exactly!

**Comment by hujinhui96** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |           x|              |            |


Good job for fullfilling all requirements and even attached a picture in your file

**Pull request closed without merging**